### PR TITLE
Web app to rest api

### PR DESCRIPTION
**Pull request by veranikaChyrun**

### CODE REVIEW

**Review by valfirst — COMMENTED** [review]
Please remove all noisy build system changes

File: docs/modules/tutorials/pages/web-app-to-rest-api.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+:plugin-short-name: web-app-to-rest-api
+
+include::plugins:partial$plugin-installation.adoc[]
+
+== Steps
+include::plugins:partial$plugin-web-app-to-rest-api-steps.adoc[]
+
+== Table Transformers
+NOTE: For more information about Table Transforrmers, please refer to xref:ROOT:glossary.adoc#_table_transformer[Table Transformers]
```
Comment (valfirst) [inline docs/modules/tutorials/pages/web-app-to-rest-api.adoc:12]: please follow the same format for headers across all docs (different ways to provide the same information may confuse users)

File: docs/modules/tutorials/pages/web-app-to-rest-api.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+==== Scenario example
+For example we need to verify that all resources by CSS selector `a` of web application `https://example.com` are valid. We know web app has sitemap with relative URL `/sitemap.xml`, but also know that some of the pages have similar redirections.
+
+Then we should add additional property:
+
+.Properties
+[source,gherkin]
+----
+web-application.main-page-ur=https://example.com
```
Comment (veranikaChyrun) [inline docs/modules/tutorials/pages/web-app-to-rest-api.adoc:27]: url

**Comment by valfirst** [comment]
Please reopen once all comments are fixed and conflicts are resolved

**Pull request closed without merging**